.github/workflows/examples-linux-standalone.yaml

```diff
@@ -57,36 +60,62 @@ jobs:
                    .environment/pigweed-venv/*.log
             - name: Build example Standalone Echo Client
               timeout-minutes: 5
-              run:
-                  scripts/examples/gn_build_example.sh examples/chip-tool
-                  out/chip_tool_debug
+              run: |
+                  scripts/examples/gn_build_example.sh examples/chip-tool out/chip_tool_debug
+                  .environment/pigweed-venv/bin/python3 scripts/tools/memory/gh_sizes.py \
+                    linux debug chip-tool \
+                    out/chip_tool_debug/chip-tool \
+                    /tmp/bloat_reports/ --collect-method readelf --by=region
```

Comment:
Why did the other yamls not need the --collect-method bits?  Might be worth a comment.